test/apps/iodemo/run_io_demo.sh

```diff
@@ -294,6 +294,11 @@ build_server_args_list() {
 			iodemo_server_args+=" $key $value"
 			shift
 			;;
+		-k)
```

Comment:
pls refactor it to use same was as in master

Reply:
done